dashboard/app/models/user.rb

```diff
@@ -471,6 +471,16 @@ def self.find_or_create_facilitator(params, invited_by_user)
   has_many :followers, through: :sections
   has_many :students, through: :followers, source: :student_user
 
+  has_many :section_instructors, foreign_key: 'instructor_id'
+  has_many :active_section_instructors, -> {where(status: :active)}, class_name: 'SectionInstructor', foreign_key: 'instructor_id'
+  has_many :sections_instructed, through: :active_section_instructors, source: :section
+
+  # TODO once the backfill creates SectionInstructors for every section owner,
```

Comment:
Could we create a JIRA task to remove this in a week or so? Just don't want it to get lost and forgotten

Reply:
Done. https://codedotorg.atlassian.net/browse/TEACH-721